### PR TITLE
SDK-1971 Add Face Capture Resource To IDV

### PR DESCRIPTION
**Pull request by BenSmithYoti**
Add Face Capture Resource to IDV with relevant tests
Add test for GeneratedProfileResponse

### CODE REVIEW

File: src/Yoti.Auth/DocScan/DocScanService.cs [hunk 1]
```diff
@@ -238,5 +302,7 @@ private static string MediaEndpoint(string sessionId, string mediaId)
         {
             return $"sessions/{sessionId}/media/{mediaId}/content";
         }
+
+        private static JsonSerializerSettings DefaultJsonSettings => new JsonSerializerSettings { NullValueHandling = NullValueHandling.Ignore };
```
Comment (echarrod) [inline src/Yoti.Auth/DocScan/DocScanService.cs:306]: Nice, could we call this `YotiDefaultJsonSettings` or something? When I saw it declared above I didn't realise it was internally defined

File: src/Yoti.Auth/DocScan/DocScanService.cs [hunk 2]
```diff
@@ -82,7 +80,7 @@ public async Task<GetSessionResult> GetSession(string sdkId, AsymmetricCipherKey
                 .WithHttpMethod(HttpMethod.Get)
                 .WithBaseUri(ApiUri)
                 .WithEndpoint(sessionEndpoint)
-                .WithQueryParam("sdkId", sdkId)
+                .WithQueryParam("sdkId", sdkId)
```
Comment (echarrod) [inline src/Yoti.Auth/DocScan/DocScanService.cs:83]: I think a bit of whitespace has crept in here. 
```suggestion
                .WithQueryParam("sdkId", sdkId)
```
Ideally we'd have a pre-commit step to trim things like this. We can add this at a later stage. 
Something like [this ](https://github.com/dotnet/format/blob/main/docs/integrations.md#pre-commitcom-hook-to-reformat)might work

**Review by echarrod — COMMENTED** [review]
Nice work Ben, just a couple of suggestions :+1:

File: src/Yoti.Auth/DocScan/DocScanService.cs (hunk 1 above)
Comment (BenSmithYoti) [reply src/Yoti.Auth/DocScan/DocScanService.cs:306]: Sure. Will amend right now for you.

File: src/Yoti.Auth/DocScan/DocScanService.cs (hunk 2 above)
Comment (BenSmithYoti) [reply src/Yoti.Auth/DocScan/DocScanService.cs:83]: Yes I saw that once PR was up on github. It's annoying as VS doesn't (with my settings anyhow - almost certainly configurable) pick up on these, but GitHub's diff viewer does. VS settings make sense for C# in VS (as that whitespace is obviously irrelevant to the MSIL and/or runtime code), GitHub needs to deal with Python etc so does it's diffs differently. Will see if I can amend my VS settings to match GitHub so we avoid going forwards. Thanks

Comment (BenSmithYoti) [reply src/Yoti.Auth/DocScan/DocScanService.cs:83]: Will amend with new commit. Thanks

**Review by echarrod — APPROVED** [review]

**Pull request merged**